server/utils/docker/utils.ts

```diff
@@ -154,15 +155,7 @@ export const removeService = async (appName: string) => {
 };
 
 export const prepareEnvironmentVariables = (env: string | null) =>
-	env
-		?.split("\n")
-		.map((line) => line.trim()) // Trim whitespace
-		.filter((line) => line && !line.startsWith("#")) // Exclude empty lines and comments
-		.map((envVar) => {
-			let [key, value] = envVar.split("=", 2);
-			value = value?.replace(/^"(.*)"$/, "$1"); // Remove surrounding double quotes
-			return `${key}=${value}`;
-		}) || [];
+	Object.entries(parse(env ?? "")).map(([key, value]) => `${key}=${value}`);
```

Comment:
A question here, what happen if the environment is invalid, they will throw an error or they will pass regardless if the env is valid or not?

Reply:
It will return an empty string if all values are incorrect. Invalid environment variables won't be included in the final result.